doc/foreign-code.rst

```diff
@@ -0,0 +1,150 @@
+******************************
+Dealing with foreign libraries
+******************************
+
+The OCaml programming language allows to interface libraries written
+in foreign languages such as C. This section explains how to do this
+with Dune. Note that it does not cover how to write the C stubs
+themselves, this is covered by the OCaml manual:
```

Comment:
I think that this should be a link. The syntax is:

    `link test <http://example.org/>`_

Reply:
thanks, I updated the link